composer.json

```diff
@@ -26,7 +27,10 @@
 	"repositories": [
 		{
 			"type": "path",
-			"url": "./packages/*"
+			"url": "./packages/*",
+			"options": {
+				"symlink": true
```

Comment:
Hm, could we remove this? Ideally, we should be symlinking by default but mirroring for production.